code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -380,11 +380,17 @@
 	return TRUE
 
 /datum/dynamic_ruleset/roundstart/revs/execute(forced = FALSE)
+	var/list/all_heads = SSjob.get_all_heads()
+	var/list/all_security = SSjob.get_all_sec()
+
+	if (all_heads.len < 4  || all_security.len < 3)
+		log_game("DYNAMIC: [ruletype] [name] failed to get enough heads or security. Refunding [cost] threat.")
+		return DYNAMIC_EXECUTE_FAILURE
```

Comment:
These requirements are super tight, should be something like at least 2 heads, at least 2 security and then a total of at least 5 from the sum of security and command imo.